### DISCUSSION TITLE
Unable to acknowledge alerts

### DISCUSSION THREAD

**Discussion by captain118** [Security-Onion-Solutions/securityonion#5845] [Unsupported Versions] [answered]
I have seen two other unresolved closed issues where people were unable to acknowledge alerts.  I too had this problem.  I tried multiple browsers, updating to the latest release.  I resulted to looking at the dev tools and seeing the traffic for the acknowledge action.  
Clicking on the acknowledgement sent me to this url:
https://<IPredacted>/#/alerts?q=%2a%20%7C%20groupby%20rule.name%20event.module%20event.severity_label&t=2021%2F10%2F06%2011%3A51%3A35%20PM%20-%202021%2F10%2F07%2011%3A51%3A35%20PM&z=America%2FNew_York&el=500&gl=500

Thats when I saw this in the dev tools network window:
![image](https://user-images.githubusercontent.com/593855/136495378-4836d5b1-64ab-4986-88a4-cfc75c51595e.png)

I was able to drilldown into an alert at that point I deleted everything after /#/alerts in the url.  After doing that I was able to continue acknowledging alerts.  But it was one at a time and at some point I had to refresh the screen and repeat..

**Comment by dougburks** [comment, marked as answer]
Since the screenshot shows a reason of `Limit of total fields [1500] has been exceeded` in the `so-playbook-alerts` index, you might want to review the Playbook alerts that you have enabled and possibly increase the field limit.
